pandas/core/common.py

```diff
@@ -2072,7 +2072,7 @@ def _random_state(state=None):
     elif isinstance(state, np.random.RandomState):
         return state
     elif state is None:
-        return np.random.RandomState()
+        return np.random.mtrand._rand
```

Comment:
Aha, I see. 
For me it does not really matter which approach we take (sklearn also uses `_rand`, so although it is private it seems ok)